### ISSUE TITLE
Upgrading to 3.1.9 breaks HTML rendering in MOTD.

### ISSUE THREAD

**Issue by bviviano** [OSC/ondemand#3875]
We just updated one of our systems form 3.1.7 -> 3.1.9.  The MOTD is no longer rendering HTML correctly.  We're using the following in `apps/dashboard/env`
```
MOTD_PATH='/etc/motd.portal'
MOTD_FORMAT='markdown_erb'
MOTD_TITLE='Announcements and Links'
```
```
# cat /etc/motd.portal 
<%= `/bin/next_pm --portal` %>
***
```
What the above looks like in 3.1.7 is
![image](https://github.com/user-attachments/assets/1a739cce-f0dc-4379-b297-a1ce27c1bfca)

In 3.1.9 I get:
![image](https://github.com/user-attachments/assets/d1224769-ea7a-493b-8b61-4ca5fcf5dbc4)

Is there a new directive or configuration option to tell MOTD to render HTML?

Thanks!

**Comment by bviviano** [comment]
Also, I am setting
```
motd_render_html: true
```
In `/etc/ood/config/ondemand.d/ondemand.yml`, with the rest of my configuration directives and I don't see anything in the latest documentation that would conflict with `motd_render_html` setting.

Thanks!

**Comment by johrstrom** [comment]
> Also, I am setting

> motd_render_html: true


Sorry about that. I'm looking over the changes and nothing associated with MOTD has changed. Setting `motd_render_html` should work for this. I'll look into it more and report back.

https://github.com/OSC/ondemand/compare/v3.1.7...v3.1.9

**Comment by bviviano** [comment]
Yeah, I was just looking back over my notes and running a diff between a 3.1.7 and 3.1.9 system, thinking I **KNOW** I've dealt with this exact problem before and reported it....

I've patched my local copy `./apps/sys/dashboard/app/models/motd_formatter/markdown_erb.rb` for 3.1.9 and retested and it's working.

**Comment by johrstrom** [comment]
Yea so sorry about all this. I have #3876 to make sure that if there's a 3.1.10 it'll be there and we can get you out of this groundhogs day.

**Comment by johrstrom** [comment]
This should be fixed in `3.1.10`, so I'm going to close this ticket.
